tests/api/test_Put.py

```diff
@@ -1,86 +0,0 @@
-import unittest
```

Comment:
Why remove this? We still need the function for the uploads of the images, no?

Reply:
Yes, we need it currently, I added it again.
However, I would move it to the same api_client.request("PUT", url, body) in an upcoming issue. This would allow us to have all endpoints except docker be handled over the urllib instead of the requests library.

Reply:
Yes, sounds good. We will have to move the docker routes soon.